proto/aggkit/prover/v1/aggchain_proof_generation.proto

```diff
@@ -20,15 +20,22 @@ message GenerateAggchainProofRequest {
   bytes l1_info_tree_leaf_hash = 4;
   // L1 Info tree proof. ([32]hash)
   repeated bytes l1_info_tree_merkle_proof = 5;
+  // L1 info tree (hash => [32]hash)
+  map<string, Proof> ger_inclusion_proofs = 6;
```

Comment:
The key should be a base64 encoded representation of the GER. Or, if we want to keep bytes, we need to fallback into the old `Entry` format.

```suggestion
  // Map of the GER with their inclusion proof. Note: the GER (string) is a base64 encoded string of the GER digest.
  map<string, Proof> ger_inclusion_proofs = 6;
```